x-pack/plugins/cases/public/components/custom_fields/schema.tsx

```diff
@@ -51,4 +41,12 @@ export const schema = {
       },
     ],
   },
+  defaultValue: {
+    label: i18n.DEFAULT_VALUE,
+    validations: [
```

Comment:
We should add a validator for the max length of the default value for the text field by using `MAX_CUSTOM_FIELD_TEXT_VALUE_LENGTH`. As this would be only for the text field we should put it in `x-pack/plugins/cases/public/components/custom_fields/text/configure.tsx`. The `UseField` accepts a configuration with validators. I think the validator about the empty field can stay here as it applies to all custom fields.